tools/generate-identifier-regex.js

```diff
@@ -3,49 +3,45 @@
 var regenerate = require('regenerate');
 
 // Which Unicode version should be used?
-var version = '7.0.0'; // note: also update `package.json` when this changes
+var version = '11.0.0'; // note: also update `package.json` when this changes
 
 // Shorthand function
 var get = function(what) {
     return require('unicode-' + version + '/' + what + '/code-points');
 };
 
-// Unicode categories needed to construct the ES5 regex
-var Lu = get('categories/Lu');
-var Ll = get('categories/Ll');
-var Lt = get('categories/Lt');
-var Lm = get('categories/Lm');
-var Lo = get('categories/Lo');
-var Nl = get('categories/Nl');
-var Mn = get('categories/Mn');
-var Mc = get('categories/Mc');
-var Nd = get('categories/Nd');
-var Pc = get('categories/Pc');
+// Get the Unicode properties needed to construct the ES6 regex.
+var ID_Start = get('Binary_Property/ID_Start');
+var ID_Continue = get('Binary_Property/ID_Continue');
+var Other_ID_Start = get('Binary_Property/Other_ID_Start');
 
-var generateES5Regex = function() { // ES 5.1
-  // http://mathiasbynens.be/notes/javascript-identifiers#valid-identifier-names
+var generateES6Regex = function() {
```

Comment:
Let’s drop the version number altogether. As of ES2015 we decided the spec would always follow the latest available Unicode version, and the ES grammar itself is unlikely to change again.